pages/data-visualization/user-manual/query-sharing.mdx

```diff
@@ -0,0 +1,110 @@
+---
```

Comment:
Is it possible to LIST all queries which have been stored for sharing?

Is it possible to EDIT an existing shared query?

Please address both questions in the doc. If "no", then explain about possible future expansion of the feature.